src/lib.rs

```diff
@@ -1,20 +1,37 @@
 #![cfg_attr(not(test), no_std)]
+
 //! Create a trusted carrier with a new lifetime that is guaranteed to be
-//! unique. When you call [`make_guard!`] to make a unique lifetime, the macro
-//! creates a [`Guard`] to hold it. This guard can be converted `into` an
-//! [`Id`], which can be stored in structures to uniquely "brand" them. A different
-//! invocation of the macro will produce a new lifetime that cannot be unified.
-//! These types have no safe way to construct them other than via
-//! [`make_guard!`].
+//! unique among other trusted carriers. When you call [`make_guard!`] to make a
+//! unique lifetime, the macro creates a [`Guard`] to hold it. This guard can be
+//! converted `into` an [`Id`], which can be stored in structures to uniquely
+//! "brand" them. A different invocation of the macro will produce a new
+//! lifetime that cannot be unified. These types have no safe way to construct
+//! them other than via [`make_guard!`] or `unsafe` code.
 //!
 //! ```rust
 //! use generativity::{Id, make_guard};
 //! struct Struct<'id>(Id<'id>);
 //! make_guard!(a);
 //! Struct(a.into());
 //! ```
+//!
+//! This is the concept of "generative" lifetime brands. `Guard` and `Id` are
+//! [invariant](https://doc.rust-lang.org/nomicon/subtyping.html#variance) over
+//! their lifetime parameter, meaning that it is never valid to substitute or
+//! otherwise coerce `Id<'a>` into `Id<'b>`, for *any* `'a` or `'b`, *including*
+//! the `'static` lifetime.
+//!
+//! Any invariant lifetime can be "trusted" to cary a brand, but when using this
```

Comment:
> Any invariant lifetime can be "trusted" to cary a brand

I don't think this is necessarily true. It's possible to create invariant lifetimes that unify. Which means they **can't** be trusted to carry a brand. That property is true of lifetimes returned by `make_guard`, but not true in general.

cary -> carry

> `Guard` and `Id` are invariant over their lifetime parameter, meaning that it is never valid to substitute or otherwise coerce `Id<'a>` into `Id<'b>`

related to the above point, it is not the invariance *itself* that gives them this property, but rather the way `generativity` implements the scoped drop guards that prevents unification. Invariance is a necessary but not sufficient condition for "generative lifetimes" to work.